main.c

```diff
@@ -77,8 +84,16 @@ static void pointer_handle_motion(void *data, struct wl_pointer *wl_pointer,
 		seat_set_outputs_dirty(seat);
 	}
 
-	seat->x = wl_fixed_to_int(surface_x) + seat->current_output->logical_geometry.x;
-	seat->y = wl_fixed_to_int(surface_y) + seat->current_output->logical_geometry.y;
+	int32_t x = wl_fixed_to_int(surface_x) + seat->current_output->logical_geometry.x;
+	int32_t y = wl_fixed_to_int(surface_y) + seat->current_output->logical_geometry.y;
+
+	if (seat->state->edit_anchor) {
+		seat->anchor_x = max(0, seat->anchor_x + (x - seat->x));
```

Comment:
I'm suspicious of this `0` here. 

Will it work with multiples monitors?
Will the selection be able to move freely between monitors? Should it?
Can't test at home, as I only have one.

Reply:
After trying the Mac feature again, I realised I didn't implement the same behavior.
If you can test both commit behavior and tell me what you think is best, I would be a grateful.